v2/anomalydetection/backend-setup.mdx

```diff
@@ -0,0 +1,1915 @@
+        security_check_response = await handle_security_checks(
+            request_id=request_id,
+            password=None,
+            brute_force_config=brute_force_config,
+            email=email,
+            phone_number=None,
+            action_type=action_type
+        )
+        if security_check_response is not None:
+            return security_check_response
+
+        # We need to call the original implementation of sign_up_post.
+        response = await original_sign_in_post(form_fields, tenant_id, api_options, user_context)
+
+        return response
+    original_implementation.sign_in_post = sign_in_post
+
+    original_generate_password_reset_token_post = original_implementation.generate_password_reset_token_post
+    async def generate_password_reset_token_post(form_fields: List[FormField], tenant_id: str, api_options: APIOptions, user_context: Dict[str, Any]):
+        request_body = await api_options.request.json()
+        if not request_body:
+            return GeneralErrorResponse(message="The request body is required")
+
+        request_id = request_body.get("requestId")
+        if not request_id:
+            return GeneralErrorResponse(message="The request ID is required")
+
+        action_type = 'send-password-reset-email'
+        ip = get_ip_from_request(api_options.request)
+        email = None
+        for field in form_fields:
+            if field.id == "email":
+                email = field.value
+        brute_force_config = get_brute_force_config(email, ip, action_type)
+
+        # we check the anomaly detection service before calling the original implementation of signUp
+        security_check_response = await handle_security_checks(
+            request_id=request_id,
+            password=None,
+            brute_force_config=brute_force_config,
+            email=email,
+            phone_number=None,
+            action_type=action_type
+        )
+        if security_check_response is not None:
+            return security_check_response
+
+        # We need to call the original implementation of sign_up_post.
+        response = await original_generate_password_reset_token_post(form_fields, tenant_id, api_options, user_context)
+
+        return response
+    original_implementation.generate_password_reset_token_post = generate_password_reset_token_post
+
+    return original_implementation
+# highlight-end
+
+init(
+    app_info=InputAppInfo(api_domain="...", app_name="...", website_domain="..."),
+    framework='...', # type: ignore
+    recipe_list=[
+        emailpassword.init(
+            # highlight-start
+            override=emailpassword.InputOverrideConfig(
+                apis=override_email_password_apis
+            )
+            # highlight-end
+        )
+    ]
+)
+```
+
+</TabItem>
+
+<TabItem value="otherFrameworks">
+other frameworks
+</TabItem>
+</BackendSDKTabs>
+
+
+### Passwordless
```

Comment:
We do not have request id in this case right? So we only have brute force detection here. No breached password detection here either! Please redo this section

Reply:
done